src/utils/backend.js

```diff
@@ -26,7 +26,17 @@ export default class Backend {
     headers: { 'Content-Type': 'application/json' },
   }));
 
-  getProfileImage = async (address) => wrapTry(async () => fetch(`${BACKEND_URL}/profile/image/${address}`));
+  getProfileImage = (address) => `${BACKEND_URL}/profile/image/${address}`;
+
+  setProfileImage = async (address, data, image = true) => wrapTry(async () => {
+    const request = {
+      method: 'post',
+      body: image ? data : JSON.stringify(data),
+    };
+    Object.assign(request, !image && { headers: { 'Content-Type': 'application/json' } });
+    console.log(request);
+    return fetch(`${BACKEND_URL}/profile/image/${address}`, request);
```

Comment:
```suggestion
    return fetch(getProfileImage(address), request);
```

Reply:
Nope, this is the `post` method where we upload it.

Reply:
Though it is a POST request, endpoints in json-apis are usually (should be) the same for get and post